libcxx/modules/CMakeLists.txt

```diff
@@ -122,7 +122,16 @@ set(LIBCXX_MODULE_STD_SOURCES
 # When that is removed the configured file can use the substitution
```

Comment:
Do we have a test for this issue? How was it found?

Reply:
We don't have a test since we don't test our module installation. It was found by trying to use modules in a bootstrap build. There the two paths are different and only the `__site_config` was available and none of our other includes.

Reply:
The goal is to add this to the existing bootstrap build, but I haven't been able to get clang-tidy to work in the bootstrap build yet. (I also need other fixes, these are already under review; we didn't have time for them yesterday.) Now that I think of of the bootstrap build will test the one module test we have in libc++. That test works without clang-tidy.